xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -1396,6 +1419,11 @@ class ATTR_DLL_LOCAL CInstanceGame : public IAddonInstance
     return static_cast<CInstanceGame*>(instance->toAddon->addonInstance)->RCResetRuntime();
   }
 
+  inline static void ADDON_FreeString(const AddonInstance_Game* instance, char* url)
```

Comment:
sorry, didn't get to update this before you cherry picked, but can you rename the parameter to 'str' or 'string' something?